format/index.md

```diff
@@ -45,18 +45,21 @@ server supports both the "bulk" and "jsonpatch" extensions.
 
 If an extension is used to form a particular request or response document,
 then it **MUST** be specified by including its name in the `ext` media type
-parameter with the `Content-Type` header. The `ext` media type parameter
-**MUST NOT** include more than one extension name.
+parameter with the `Content-Type` header. 
+The value of the `ext` media type parameter 
+**MUST** be a comma-separated (U+002C COMMA, ",") list of extension names.
 
 For example: a response that includes the header `Content-Type:
-application/vnd.api+json; ext=jsonpatch; supported-ext=bulk,jsonpatch`
-indicates that the document is formatted according to the "jsonpatch"
-extension.
+application/vnd.api+json; ext=bulk,jsonpatch; supported-ext=bulk,jsonpatch`
+indicates that the document is formatted according to the "bulk" and "jsonpatch"
+extensions.
```

Comment:
Since bulk and jsonpatch can't be combined, let's leave the example as it was OR use completely fictitious names as @bintoro suggested. For instance, `ext=a,b; supported-ext=a,b,c,d`.